server/server.go

```diff
@@ -159,7 +159,8 @@ func (server *Server) getDatabaseConfig() (string, string, bool, bool) {
 	}
 	databaseDropOnCreate := config.GetBool("database/drop_on_create", false)
 	databaseCascade := config.GetBool("database/cascade_delete", false)
-	return databaseType, databaseConnection, databaseDropOnCreate, databaseCascade
+	databaseAutoMigrate := config.GetBool("database/auto_migrate", true)
```

Comment:
1. `no_init` it lacks a sanity check, you will notice that db is in wrong version by crashing requests
1. `auto_migrate` default `true` is set to preserve current behavior, @nati do you have an opinion on that front?